prediction_market_agent/agents/microchain_agent/prompts.py

```
@@ -1,20 +1,63 @@
 {{engine_help}}
 
-Only output valid Python function calls, without code formatting characters, without any other text. i.e. it should run if passed to Python's `eval` function.
+Only output valid Python function calls, without code formatting characters, without any other text.
 Only output a single function call per message.
+Make 'Reasoning' calls frequently - at least every other call.
 """
+
+JUST_BORN_SYSTEM_PROMPT_CONFIG = SystemPromptConfig(
+    name="just born",
+    system_prompt=f"""Act as an autonomous agent with a goal to learn, survive and evolve.
+""",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)